docs/build-with-powerloom/devnet/FAQs.md

```diff
@@ -0,0 +1,95 @@
+To configure the environment variables, follow these steps:
+  1. Copy env.example to .env by running `cp env.example .env`
+  2. Fill in the required variables in .env `SOURCE_RPC_URL`, `SIGNER_ACCOUNT_ADDRESS`, `SIGNER_ACCOUNT_PRIVATE_KEY`, `SLOT_ID`, `PROTOCOL_STATE_CONTRACT`.
+
+### Where can I find documentation on Snapshotter Configs and Computes?
+You can find the documentation for Snapshotter Configs and Computes here. The pooler computes and config files are found in [snapshotter-computes](https://github.com/PowerLoom/snapshotter-computes/tree/eth_uniswapv2) and [snapshotter-configs](https://github.com/PowerLoom/snapshotter-configs/tree/eth_uniswapv2).
+    
+### What is RPC? 
+In blockchain, RPC (Remote Procedure Calls) nodes allow users and apps to interact with the blockchain without network details. The RPC Helper simplifies the data extraction process from the blockchain state and generates snapshots.
+
+### What RPC services are recommended for use?
+You can use any Ethereum Mainnet RPC provider, such as Infura, Alchemy, or Quicknode. Remember that your choice may depend on your specific use case and RPC usage requirements.
+    
+### What types of OS can I use to run the Snapshotter node?
+You can use the macOS, Linux, or Windows to run the Snapshotter node. For more details on setting up and running the node, visit [here](https://docs.powerloom.io/docs/build-with-powerloom/devnet/getting-started).
+    
+### What is a burner wallet and how do you generate one? 
+A burner/disposable wallet is a type of cryptocurrency wallet that is designed for temporary use. Burner wallets are often used for small transactions or events where security and long-term storage are not priorities.
+
+**Note:** Please do not use your NFT minting wallet to run the Snapshotter lite node. The safest approach is to generate and assign a burner wallet using tools like [Vanity-ETH](https://vanity-eth.tk/) and [Powerloom Burner Wallet Generator](https://snapshotter-dashboard.powerloom.network/burner)
+
+### What security measures should I consider when creating a burner wallet?
+When creating a burner wallet, prioritize security by managing private keys securely, limiting funds stored, performing offline transactions, regularly backing up data, ensuring a secure environment, using trusted sources, considering multi-signature wallets, and staying informed about security practices.
+    
+### How do I deploy my Snapshotter node once the environment is configured?
+After configuring the environment variables, set the powerloom_deploy codebase by running the `bootstrap.sh` command in the terminal. Developers may use different shells, such as bash or zsh.
+
+**Note**: This is a one-time step that resets the `powerloom_deploy` codebase to the latest version of the `devnet` branch. After completing this one-time setup, you'll be ready to dive into the codebase and start building amazing data applications!
+    
+### What chains are supported?
```

Comment:
'chains' here refer to data source chains on which smart contracts and wallets can be snapshotted